### PR TITLE
[DOCS] Replace Slack alerts with email alerts

### PR DESCRIPTION
**Pull request by josectobar**
We're removing Slack alerts from GX Cloud. I'm replacing the documentation with email alerts.

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [x] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/docs/cloud/alerts/manage_alerts.md [hunk 1]
```diff
@@ -4,60 +4,29 @@ title: 'Manage alerts'
 description: Create and manage alerts in GX Cloud.
 ---
 
-Add alerts to your GX Cloud workflows to receive Slack notifications when a Checkpoint is successful, it fails, or when it succeeds or fails.
+Stay informed about the health of the Data Assets that you care about by enabling alerts in GX Cloud.
```
Comment (AkshatAtGX) [inline docs/docusaurus/docs/cloud/alerts/manage_alerts.md:7]: Recommend changing this to:

> _You are automatically subscribed to email alerts for any Data Asset you create in GX Cloud. An email will be sent whenever the Data Asset fails validation._

File: docs/docusaurus/docs/cloud/alerts/manage_alerts.md [hunk 2]
```diff
@@ -4,60 +4,29 @@ title: 'Manage alerts'
...
-6. Complete the following fields:
-
-    - **Alert name:** Enter a meaningful name for the alert.
-
-    - **Webhook:** Enter the Slack incoming webhook URL. For more information about Slack incoming webhooks, see [Sending messages using incoming webhooks](https://api.slack.com/messaging/webhooks).
-
-    - **Alert type:** Select the type of alert to send to Slack. **Show all** sends a notification when the Checkpoint succeeds or fails, **Failures only** sends a notification when the Checkpoint fails, and **Success only** sends a notification when the Checkpoint succeeds. 
-
-7. Click **Save**.
+When you add a new Data Asset in GX Cloud, you are automatically subscribed to receive email alerts. If a Data Asset validation fails, an email will be sent to the address associated with your GX Cloud account.
```
Comment (AkshatAtGX) [inline docs/docusaurus/docs/cloud/alerts/manage_alerts.md]: Recommend changing this to:

> _You are automatically subscribed to email alerts for any Data Asset you create in GX Cloud. An email will be sent whenever the Data Asset fails validation._

File: docs/docusaurus/docs/cloud/alerts/manage_alerts.md [hunk 3]
```diff
@@ -4,60 +4,29 @@ title: 'Manage alerts'
...
-2. In the **Data Assets** list, click the Data Asset name.
-
-3. Click the **Expectations** tab and then click **Alerts**.
-
-4. Click **Delete** for the alert you want to delete.
+## Delete an email alert
 
-5. Click **OK**.
\ No newline at end of file
+Email alerts are deleted automatically when you delete the user or a Data Asset associated with the alert.
```
Comment (NathanFarmer) [inline docs/docusaurus/docs/cloud/alerts/manage_alerts.md]: Do we need this section? I'm not sure if this is really relevant to the user. If the alert is deleted, we still retain any user information such as email.

Comment (josectobar) [reply docs/docusaurus/docs/cloud/alerts/manage_alerts.md]: Yeah, I see your point. I can remove it.

**Review by NathanFarmer — APPROVED** [review]

**Pull request merged**